httpserver/http_connection.cpp

```diff
@@ -501,27 +505,27 @@ void connection_t::process_request() {
 
     const auto target = request_.target();
     switch (request_.method()) {
-    case http::verb::post:
-        {
-            std::string reason;
-            if (!service_node_.snode_ready(reason)) {
-                LOKI_LOG(debug, "Ignoring post request; storage server not ready: {}", reason);
-                response_.result(http::status::service_unavailable);
-                body_stream_ << fmt::format("Service node is not ready: {}\n", reason);
-                break;
-            }
+    case http::verb::post: {
+        std::string reason;
+        if (!service_node_.snode_ready(reason)) {
+            LOKI_LOG(debug,
+                     "Ignoring post request; storage server not ready: {}",
+                     reason);
+            response_.result(http::status::service_unavailable);
+            body_stream_ << fmt::format("Service node is not ready: {}\n",
+                                        reason);
+            break;
         }
         if (target == "/storage_rpc/v1") {
             /// Store/load from clients
-            LOKI_LOG(trace, "got /storage_rpc/v1");
+            LOKI_LOG(trace, "POST /storage_rpc/v1");
 
             try {
                 process_client_req();
             } catch (std::exception& e) {
+                this->body_stream_ << fmt::format("exception caught while processing client request: {}", e.what());
```

Comment:
Nit pick but if we are already tidying up the logs/body responses I would prefer leading capital letters here and below

Reply:
I only copied the message that we were already printing to log. Updated to use capital letters in this particular case now. Can't possibly make everything perfect in this one PR, I'm sure we will be coming back to improving logs in the future 😉